docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3'
+services: 
+  cypress-tests:
+    build: .
+    image: dhis2/e2e-cy
+    shm_size: '2gb'
+    environment:
```

Comment:
There's no `CYPRESS_REPORT_PORTAL_ENABLED` env var here. 🤔  Is there a way to disable the Report Portal?

Reply:
No, i'll fix readme. The plugin we use doesn't support that, so if you wanted to implement it you could modify the reporter-config (remove @reportportal/agent-js-cypress from reporterEnabled) in docker-entrypoint or smth. Currently, you can skip reporting if you run `cy:parallel` or `cy:test`. And `cy:parallel` will always report to report portal.